modules/qt/hm.nix

```diff
@@ -39,7 +39,9 @@
 
       recommendedStyles = {
         gnome = if config.stylix.polarity == "dark" then "adwaita-dark" else "adwaita";
+        #NOTE: setting a style for kde might not be desirable as user may want to change it via UI
         kde = "breeze";
+        kde6 = "breeze";
```

Comment:
I assume this is unrelated to the fix.

Reply:
This is something that I wanted to receive feedback on. Because technically I'd either have to add kde6 or remove kde to have some consistency.